### PR TITLE
Upgrade funcunit to syn 0.5.0

### PR DESCRIPTION
**Pull request by kdillon**
Upgrades funcunit to syn 0.5.0, which brings with it...

- Universal PointerEvent support (minus rightclick)
- Universal TouchEvent support (minus rightclick)
- Expanded Modifier Key support
- Modifier Key fixes to Ctrl
- Many other bugsfixes, doc, and expanded tests

### CODE REVIEW

**Comment by matthewp** [comment]
:D

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
   "scripts": {
     "preversion": "npm test && npm run build",
     "version": "git commit -am \"Update dist for release\" && git checkout -b release && git add -f dist/",
-    "postversion": "git push --tags && git checkout master && git branch -D release && git push",
+    "postpublish": "git push --tags && git push",
```
Comment (phillipskevin) [inline package.json]: any reason you want to keep / push the release branch?

Comment (phillipskevin) [inline package.json]: I think this needs to be a little different than Syn since the `version` command creates a release branch and adds the built code.

Comment (phillipskevin) [inline package.json]: You should just keep what the script used to do and change it to `postpublish`.

**Pull request merged**